dev/io.openliberty.checkpoint/resources/io/openliberty/checkpoint/resources/CheckpointMessages.nlsprops

```diff
@@ -23,7 +23,7 @@
 # MSG_DESCRIPTIVE_NAME_CWWKC0000.explanation=Explanation text for the message
 # MSG_DESCRIPTIVE_NAME_CWWKC0000.useraction=User action text for the message
 
-CHECKPOINT_DUMP_INITIATED_CWWKC0451=CWWKC0451I: A server checkpoint was requested. When the checkpoint completes, the server stops.
+CHECKPOINT_DUMP_INITIATED_CWWKC0451=CWWKC0451I: A server checkpoint "{0}" was requested. When the checkpoint completes, the server stops.
```

Comment:
The usual format is "A _variable_ _noun_". Unless the message will be less understandable. Thus, try:

A "{0}" server checkpoint

Reply:
After discussion with @sebratton, opted to not change the `A server checkpoint "{0}"`  format.